CHANGELOG.md

```diff
@@ -366,6 +402,19 @@ Branch: `nilrt/22.5/hardknott`
 - [Removed](https://github.com/ni/meta-nilrt/pull/290) packages dropped from upstream.
 
 
+----
+## 8.20
+Branch: `nilrt/25.0/sumo`
+
+### nilrt
+
+No changes.
+
+### meta-nilrt
+
+- [Added](https://github.com/ni/meta-nilrt/commit/c5aca13f9f69170f0b21b05ea5adde77253f7279) new signing key into opkg-keyrings.
```

Comment:
Why commit url instead of PR like others?

Reply:
You used text ".. a new signing key to the base system image" for scarthgap.. why not use same for here?

Reply:
Because this one was a cherry-pick.

Reply:
I have added the PR link.